views/layouts/main.handlebars

```diff
@@ -23,5 +24,6 @@
 
         {{!-- this file is for the button handling in the NewOrder modal --}}
         <script src="/js/newOrder.js"></script>
```

Comment:
These should not be in the main... they should be in the page they’re clicked

Reply:
Hi Taher, if i understand correctly, the PR comment mentioned 'newOrder.js' should not be in the main.handlebars; but instead, in the associated handlebars (i.e: newOrders.handlebars)
I take a look at wk 13 MVC , Day 3, class activities #17, the CaApp example. Below; it shows the cat.js is not defined under cat.handlebars - but instead; within main.handlebars.....
I originally thought the same way you did, but then, the <head> is only defined in main.handlebars; so may be that's why in catApp ex,
*.js are defined under main.handlebars ?....what do you think...

Reply:
tested in newInventory.js and not in main.handlebar; page did not come up.
agree so far to leave it on main.handlebars until further testing.